packages/core/package.json

```diff
@@ -5,8 +5,28 @@
   "license": "MIT",
   "main": "dist/use-gesture-core.cjs.js",
   "module": "dist/use-gesture-core.esm.js",
+  "exports": {
```

Comment:
Ye, so this looks OK-ish now - but we still end up with extensionless imports in the generated `.d.ts`. That has to be fixed in preconstruct itself.